//***********************************************************************************
//Discription: this head file define the basic widget of GUI and a paint event.
//***********************************************************************************

#include "datalayer.hh"
#include <QDialog>
#include <QtGui>
#include <QtCore>
#include <QWidget>

